README.md

```diff
@@ -94,13 +106,9 @@ if err != nil {
 }
 ```
 
-### Supported data types
-- bool
-- uint8
-- uint16
-- uint32
-- uint64
-- slice
-- array
-- struct
```

Comment:
I think we need these.

Reply:
I relocated this information to directly under the 'Usage examples' header:

**Notice:** SSZ supports `bool`, `uint8`, `uint16`, `uint32`, `uint64`, `slice`, `array`, `struct` and `pointer`data types.

If you'd like me to present it a different way or perhaps specify a little clearer, happy to modify it.